<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CALC</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Sans Serif&quot;;
background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="press_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
font: italic 9pt &quot;Sans Serif&quot;;
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>561</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;
</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_11">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>รท</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_13">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_15">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="press_button_16">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>one
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 255, 178);
border-color: rgb(255, 255, 255);
selection-color: rgb(255, 255, 255);
border-radius: 10px;
font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: italic 18pt &quot;Hack&quot;;</string>
   </property>
   <property name="text">
    <string>BUILD WITH QT BY THAMARAI</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
